data/harisdev-netizen.json

```
@@ -37,6 +37,11 @@
       "url": "https://www.linkedin.com/in/hariskhan00/",
       "icon": "FaLinkedin"
     }
```

```suggestion
    },
```
There is comma missing that's why the tests are failing.